requirements/test.txt

```diff
@@ -5,5 +5,6 @@ fake-factory==0.5.7
 coverage
 mock
 pytest
+mixer
```

Comment:
Yeah, in the code I'm using right now. That line just got committed early
(this PR isn't finished too, have to fix the tests first)
On Jul 2, 2016 4:55 PM, "Jamie Alexandre" <notifications@github.com> wrote:

> In requirements/test.txt
> <https://github.com/learningequality/kolibri/pull/196#discussion_r69383879>
> :
>
> > @@ -5,5 +5,6 @@ fake-factory==0.5.7
> >  coverage
> >  mock
> >  pytest
> > +mixer
>
> Is this getting used?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learningequality/kolibri/pull/196/files/4c1d2fd78b5d5224b710e16a202eaa7bf8fbd9e4#r69383879>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALt05hu83CvpCIubVE8XsimF0d5x7GEks5qRvpagaJpZM4JDTji>
> .
>